Mathlib/NumberTheory/FLT/Three.lean

```diff
@@ -16,6 +18,26 @@ The goal of this file is to prove Fermat Last theorem in the case `n = 3`.
 
 ## TODO
 Prove case 2.
+
+## Implementation details
+We follow the proof in https://webusers.imj-prg.fr/~marc.hindry/Cours-arith.pdf, page 43. The
+strategy is the following:
+* Case 1 is completely elementary and is proved using congruences modulo `9`.
+* To prove case 2, we consider the generalized equation `a ^ 3 + b ^ 3 = u * c ^ 3`, where `a`, `b`,
```

Comment:
After reading through the rest of the file... Is this not just `Associated (a ^ 3 + b ^ 3) (c ^ 3)`? (Perhaps we defer checking whether that's an easier formulation after you've landed everything, though.)

Reply:
Yes, what we prove is that `a ^3 + b ^ 3` and `c ^ 3` cannot be associated in `𝓞 K`. I have the feeling we don't gain anything in using `Associated`: in all proofs we would introduce immediately the unit.